### DISCUSSION TITLE
Group by [Jul 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6045] [Discord Threads]
> 🙋‍♂️ **dandiscrda**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@1094262160491823164> not in immediate road map  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1115293667746381854)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
